back/app/services/side_fx_project_service.rb

```diff
@@ -77,6 +77,26 @@ def after_delete_inputs(project, user)
     LogActivityJob.perform_later project, 'inputs_deleted', user, Time.now.to_i
   end
 
+  def after_votes_by_user_xlsx(project, user)
+    LogActivityJob.perform_later(
+      project,
+      'exported_votes_by_user',
+      user,
+      project.updated_at.to_i,
+      project_id: project.id
+    )
+  end
+
+  def after_votes_by_input_xlsx(project, user)
+    LogActivityJob.perform_later(
+      project,
+      'exported_votes_by_input',
+      user,
+      project.updated_at.to_i,
```

Comment:
This param is `acted_at`. Do we update `project` before logging this activity? If not, maybe this could work:
```suggestion
      Time.now.to_i,
```
And the same question about `after_votes_by_user_xlsx`.

Reply:
Good catch!